docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -469,6 +469,28 @@ By default, when a Quarkus application starts (in regular or dev mode), it will
 by setting the `-Dquarkus.banner.enabled=false` Java System Property, or by setting the `QUARKUS_BANNER_ENABLED` environment variable to `false`.
 Furthermore, users can supply a custom banner by placing the banner file in `src/main/resources` and configuring `quarkus.banner.path=name-of-file` in `application.properties`.
 
+== Non Application endpoints
+
+Various Quarkus extensions contribute non-application endpoints that provide different kinds of information about the application.
+Examples of such extensions are the xref:smallrye-health.adoc[health], xref:micrometer.adoc[metrics], OpenAPI and info extensions.
```

Comment:
```suggestion
Examples of such extensions are the xref:smallrye-health.adoc[health], xref:micrometer.adoc[metrics], xref:openapi-swaggerui.adoc[OpenAPI] and info extensions.
```

Reply:
Ah nice, I tried to find that one real quick but missed it :)

Reply:
Applied, thanks